support/ebpf/integration_test.ebpf.c

```diff
@@ -1,15 +1,16 @@
 // This file contains the code and map definitions that are used in integration tests only.
 
 #include "bpfdefs.h"
-#include "frametypes.h"
-#include "types.h"
 #include "extmaps.h"
+#include "frametypes.h"
 #include "tracemgmt.h"
+#include "types.h"
 
 extern bpf_map_def kernel_stackmap;
 
-static inline __attribute__((__always_inline__))
-void send_sample_traces(void *ctx, u64 pid, s32 kstack) {
+static inline __attribute__((__always_inline__)) void
+send_sample_traces(void *ctx, u64 pid, s32 kstack)
```

Comment:
I think there is a switch for clang-format to enforce a format like here, just for consistency.

Reply:
It would have been `AlwaysBreakAfterReturnType`, but I retreat from this, it doesn't always make code more readable.